commit abb86adf7f749b3d44887d28bc96b43c5a1e0631
Author: Ben Gamari <bgamari.foss@gmail.com>
Date:   Thu Mar 6 21:22:28 2014 +0100

    config.mk.in: ARM now supports dynamic linking with the LLVM backend
    
    Signed-off-by: Austin Seipp <austin@well-typed.com>
    (cherry picked from commit d574fcbba09fd6c9d10a79e19daf5f15bb0a6cde)

	Modified   mk/config.mk.in
diff --git a/mk/config.mk.in b/mk/config.mk.in
index 2c997de..fef1fb8 100644
--- a/mk/config.mk.in
+++ b/mk/config.mk.in
@@ -95,8 +95,7 @@ TargetElf = YES
 endif
 
 # Some platforms don't support shared libraries
-NoSharedLibsPlatformList = arm-unknown-linux \
-	powerpc-unknown-linux \
+NoSharedLibsPlatformList = powerpc-unknown-linux \
 	x86_64-unknown-mingw32 \
 	i386-unknown-mingw32 \
 	sparc-sun-solaris2 \
